### ISSUE TITLE
About weight init

### ISSUE THREAD

**Issue by mutianxu** [HuguesTHOMAS/KPConv-PyTorch#50]
Hi @HuguesTHOMAS ,

Did you add the weight init of the bias or weight in **MLP** or **BN** in your code?

Will this influence the performance?

Thanks~

**Comment by HuguesTHOMAS** [comment]
Hi @mutianxu,

Do you mean did I try to use a different weight initialization method for my `UnaryBlock `and `BatchNormBlock`?

No I did not so I dont know how it would influence the performances, but I believe it would not. If you try that, dont hesitate to post the results, for other people to know.

Best,
Hugues

**Comment by mutianxu** [comment]
@HuguesTHOMAS ,
Yes, that is what I mean.

I will try it and post the result.

Thanks a lot.
